### DISCUSSION TITLE
Table schema for fixed-width tabular files

### DISCUSSION THREAD

**Discussion by peterdesmet** [frictionlessdata/frictionlessdata.io#625] [Q&A] [answered]
👋 I'm dealing with an archive of tabular data that is organized as fixed-width text files:

```
# vol2bird Vertical Profile of Birds (VPB)
# source: WMO:06475,RAD:BX43,PLC:Helchteren,NOD:behel,CTY:605,CMT:behel_scan_200km_dp_dBZ
# polar volume input: /tmp/20191015143000.rad.behel.pvol.dbzh.scanz.pvol.h5
# date   time HGHT    u      v       w     ff    dd  sd_vvp gap dbz     eta   dens   DBZH   n   n_dbz n_all n_dbz_all
20191015 1430    0    nan    nan     nan   nan   nan   0.69 T    nan    nan   0.00  23.63     0     0 10276 10411
20191015 1430  200    nan    nan     nan   nan   nan   0.92 T    nan    nan   0.00  23.18     0     0 60511 61162
20191015 1430  400    nan    nan     nan   nan   nan   0.77 T    nan    nan   0.00  18.47     0     0 22723 22739
```

I would like to describe this columns in a Table Schema. Is there a way in [Tabular Data Resource](https://specs.frictionlessdata.io/tabular-data-resource/) to express fixed-width tabular data? E.g. a property to define the start/end positions of the columns?

**Comment by roll** [comment, marked as answer]
Hi @peterdesmet,

Not yet. We have a feature request for the framework to support it - https://github.com/frictionlessdata/frictionless-py/issues/476
